txn_writer.go

```diff
@@ -21,15 +21,63 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLE (these lines may be deleted)
+#
+## "required" lists a set of packages (not projects) that must be included in
```

Comment:
What I was suggesting in the previous round of comments was that the docs should be after the thing they document, which I know is different to how it works in Go code, but in this case we're looking at a config example so I think it makes some kind of sense.